java/src/jmri/jmrit/operations/rollingstock/RollingStock.java

```diff
@@ -813,7 +815,7 @@ public void propertyChange(java.beans.PropertyChangeEvent e) {
                         if (e.getNewValue() != null) {
                             Location newLocation = locationManager.getLocationByReporter((jmri.Reporter) e.getNewValue());
                             if (newLocation != getLocation())
-                                setLocation(newLocation, null);
+                                setWhereLastSeen(newLocation);
```

Comment:
Dan, 

1) Why are you duplicating the When Last Seen and Where Last Seen properties of the ID Tag?  Those should be persistent values associated with the ID tag, so we can look those up, and shouldn't need to be stored with the rolling stock.

2) The code that sets Where Last Seen and When Last Seen instead of the Location and the Date also need to store the Location and the Date.  We may not have ID tag readers at ever location, but when a car passes a reader, that reader is associated with a location, and we know the car MUST be at that location.

The 2nd part of this is VERY important to me, because it changes the previous behaviour.  This code change breaks scripts that I've got locally to print wheel reports.

Paul